optimade/filtertransformers/mongo.py

```diff
@@ -201,27 +200,30 @@ def fuzzy_string_op_rhs(self, arg):
 
         if arg[0] == "CONTAINS":
```

Comment:
If I understand correctly this logic (returning with the most common case) I would suggest this way:
```
        if arg[0] == "STARTS":
            return {"$regex": f"^{pattern}"}
        elif arg[0] == "ENDS":
            return {"$regex": f"{pattern}$"}
        
        # CONTAINS
        return {"$regex": f"{pattern}"}
```